### PR TITLE
Fix playbooks/openshift-master/library move to symlink

### PR DESCRIPTION
**Pull request by sdodson**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1336647

### CODE REVIEW

**Comment by sdodson** [comment]
@tdawson Any thoughts on simplifying packaging? roles and playbooks are very intertwined. I'd almost prefer just shipping one 'openshift-ansible' and perhaps an 'openshift-ansible-docs' package if anything else. I found several dangling symlinks because we've sliced packaging up so much we lost some content.

**Comment by detiber** [comment]
So, think the intertwinedness is something that we can address with the Ansible 2.0+ migration.

We'll be able to more easily embed filter plugins directly into roles, rather than having to have them defined at the top level and the playbooks themselves will contain less logic in them.

I like the idea of keeping the roles as a separate deliverable, since it allows for a way to consume the officially released roles with Ansible Tower without having to pull in the playbooks as well.

**Comment by sdodson** [comment]
I just realized the dangling symlinks isn't an issue because the ansible.cfg loads them from another path. I've removed that change. Good to review now.

File: openshift-ansible.spec [hunk 1]
```diff
@@ -78,6 +79,8 @@ popd
 %files
 %doc LICENSE.md README*
 %dir %{_datadir}/ansible/%{name}
+%{_datadir}/ansible/%{name}/library
+%ghost %%{_datadir}/ansible/%{name}/playbooks/common/openshift-master/library.rpmmoved
```
Comment (tdawson) [inline openshift-ansible.spec]: typo - one too many % before datadir, should be  
+%ghost %{_datadir}/ansible/%{name}/playbooks/common/openshift-master/library.rpmmoved

**Comment by tdawson** [comment]
It's a little wordy, but I think it's very good.  I would much rather have comments and documentation showing why something like that is put in, than not having any explanation.

**Comment by sdodson** [comment]
@tdawson thanks, fixed now

**Comment by sdodson** [comment]
@detiber ready to go CI jobs don't build rpms currently so not running them

**Comment by detiber** [comment]
lgtm

**Pull request merged**